tests/test_inference/test_api_security.py

```
@@ -0,0 +1,181 @@
+# ---------------------------------------------------------------------------
+# Test 1: /health — 민감 정보 미노출
+# ---------------------------------------------------------------------------
+
+
+def test_health_check(mock_manager):
+    """health 엔드포인트가 민감 정보를 노출하지 않고 인덱스 상태를 반환한다."""
+    _SENSITIVE_PATH = "/home/user/models/secret-model"
+    mock_model = MagicMock()
+    mock_model.model_path = _SENSITIVE_PATH
+    with (
+        patch("src.inference.api_server.runtime_config.paths.local_docs_root", "/tmp/local-docs"),
```

Ruff (0.15.9)

[error] Probable insecure usage of temporary file or directory: "/tmp/local-docs"

(S108)